core/src/main/java/edu/stanford/futuredata/macrobase/contrib/aria/APICubeResult.java

```diff
@@ -0,0 +1,10 @@
+package edu.stanford.futuredata.macrobase.contrib.aria;
```

Comment:
kind of strange this lives in `core` even though it's really `contrib`... do we want to rethink here, or just let core contain basically everything